docs/CONTRIBUTING.md

```diff
@@ -249,3 +250,48 @@ These contribution guidelines were inspired by and are based on Atom’s
 contribution guidelines. They were modified for the purposes of this repository.
 https://github.com/atom/atom/blob/master/CONTRIBUTING.md - Copyright (c)
 2011-2017 GitHub Inc. (MIT)
+
+### JSDoc Styleguide
+
+For code documentation, we use [JSDoc](http://usejsdoc.org/).
+You can generate a HTML documentation out of it.
+Navigate to the projects folder and run:
+
+```
+npm run docs:build
+```
+
+To access the documentation via browser, use the following command:
+
+```
+npm run docs:serve
+```
+
+If you add or change code, please always provide or update the corresponding code documentation.
+
+**Always mandatory:** A general description of what the code is doing.
+If tags follow, leave a blank line between the description and the tags.
+
+**Tags to use:**
+
+| Tag          | When to use                                                             |
+| ------------ | ----------------------------------------------------------------------- |
+| `@class`     | Describes the constructor of a class.                                   |
+| `@event`     | Describes an event, that the app is listening to.                       |
+| `@example`   | If convenient, provide an example.                                      |
+| `@func`      | Only if JSDoc does not recognize the code as a function, but it is one. |
+| `@link`      | To add inline links.                                                    |
```

Comment:
To what? Other documentation on the site or external URLs?

Reply:
both. Will add that to make it clear.